core/server/master/src/main/java/alluxio/master/file/contexts/OperationContext.java

```diff
@@ -22,14 +22,31 @@
 public class OperationContext<T extends com.google.protobuf.GeneratedMessageV3.Builder<?>> {
   // Proto message that is being wrapped
   private T mOptionsBuilder;
+  // Used to track client call status.
+  private CallTracker mCallTracker;
 
   /**
    * Creates an instance with given proto message.
    *
    * @param optionsBuilder Internal proto message builder instance
    */
   public OperationContext(T optionsBuilder) {
+    this(optionsBuilder, null);
     mOptionsBuilder = optionsBuilder;
+    mCallTracker = () -> {
```

Comment:
can we make this a static/constant noop tracker, to avoid re-creating it frequently?